lib/iris/tests/test_grib_load_translations.py

```diff
@@ -476,80 +412,5 @@ def test_grib1_known_standard_params(self):
-        grib = self.mock_grib()
-        grib.discipline = discipline
-        grib.parameterCategory = category
-        grib.parameterNumber = param
-        cube = self.cube_from_message(grib)
-        try:
-            _cf_units = cf_units.Unit(units_str)
-        except ValueError:
-            _cf_units = cf_units.Unit('???')
-        self.assertEqual(cube.standard_name, standard_name)
-        self.assertEqual(cube.long_name, long_name)
-        self.assertEqual(cube.units, _cf_units)
-
-    def test_grib2_unknownparam(self):
-        grib = self.mock_grib()
-        grib.discipline = 999
-        grib.parameterCategory = 999
-        grib.parameterNumber = 9999
-        cube = self.cube_from_message(grib)
-        self.assertEqual(cube.standard_name, None)
-        self.assertEqual(cube.long_name, None)
-        self.assertEqual(cube.units, cf_units.Unit("???"))
-
-    def test_grib2_known_standard_params(self):
-        # check we know how to translate at least these params
-        # I.E. all the ones the older scheme provided.
-        full_set = [
-            (0, 0, 0, "air_temperature", None, "kelvin"),
-            (0, 0, 2, "air_potential_temperature", None, "K"),
-            (0, 1, 0, "specific_humidity", None, "kg kg-1"),
-            (0, 1, 1, "relative_humidity", None, "%"),
-            (0, 1, 3, None, "precipitable_water", "kg m-2"),
-            (0, 1, 22, None, "cloud_mixing_ratio", "kg kg-1"),
-            (0, 1, 13, "liquid_water_content_of_surface_snow", None, "kg m-2"),
-            (0, 2, 1, "wind_speed", None, "m s-1"),
-            (0, 2, 2, "x_wind", None, "m s-1"),
-            (0, 2, 3, "y_wind", None, "m s-1"),
-            (0, 2, 8, "lagrangian_tendency_of_air_pressure", None, "Pa s-1"),
-            (0, 2, 10, "atmosphere_absolute_vorticity", None, "s-1"),
-            (0, 3, 0, "air_pressure", None, "Pa"),
-            (0, 3, 1, "air_pressure_at_sea_level", None, "Pa"),
-            (0, 3, 3, None, "icao_standard_atmosphere_reference_height", "m"),
-            (0, 3, 5, "geopotential_height", None, "m"),
-            (0, 3, 9, "geopotential_height_anomaly", None, "m"),
-            (0, 6, 1, "cloud_area_fraction", None, "%"),
-            (0, 6, 6, "atmosphere_mass_content_of_cloud_liquid_water", None,
-                "kg m-2"),
-            (0, 7, 6,
-             "atmosphere_specific_convective_available_potential_energy",
-             None, "J kg-1"),
-            (0, 7, 7, None, "convective_inhibition", "J kg-1"),
-            (0, 7, 8, None, "storm_relative_helicity", "J kg-1"),
-            (0, 14, 0, "atmosphere_mole_content_of_ozone", None, "Dobson"),
-            (2, 0, 0, "land_area_fraction", None, "1"),
-            (10, 2, 0, "sea_ice_area_fraction", None, "1")]
-
-        for (discipline, category, number,
-             standard_name, long_name, units) in full_set:
-            self.known_grib2(discipline, category, number,
-                             standard_name, long_name, units)
-
-
```

Comment:
Removed these tests as they are for grib edition 2 but the tests uses GribWrapper which now only works for grib edition 1.